test/unit/mongo/MongoTripodTest.php

```diff
@@ -585,17 +585,17 @@ public function testReadPreferencesOverMultipleSaves(){
             ->method('getDataUpdater')
             ->will($this->returnValue($tripodUpdate));
 
-        $expectedCollectionReadPreference = $tripodMock->collection->getReadPreference();
-        $expectedDbReadPreference = $tripodMock->db->getReadPreference();
+        $expectedCollectionReadPreference = $tripodMock->getCollectionReadPreference();
+//        $expectedDbReadPreference = $tripodMock->db->getReadPreference();
```

Comment:
They're commented because right now there's not a way to get at the database to even check the read preferences.  At least, not from the test suite.  It's TBD whether or not we that's important and what would be required to make that happen.